builder/steps/gen-dockerfile/contents/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "nodejs-gen-dockerfile",
+  "description": "",
+  "scripts": {
+    "pretest": "gulp",
+    "test": "gulp check-format && mocha dist/test/**/*.js",
+    "prestart": "gulp",
+    "start": "node dist/src/main.js"
+  },
+  "version": "0.0.1",
+  "devDependencies": {
```

Comment:
Can you open an issue to track this to make sure this is tracked (unless you already have a solution as part of this PR).